common/modifiers/01_ancient_magic_modifiers.txt

```diff
@@ -1,25 +1,25 @@
 ﻿glimpse_the_future_spell_modifier = {
 	icon = martial_positive
-	cultural_head_fascination_mult = 0.2
+	cultural_head_fascination_mult = 0.05
 	stacking = yes
 }
 
 financial_times_spell_modifier = {
 	icon = martial_positive
-	monthly_income_mult = 0.1
```

Comment:
oh, yeah this, when one stacked like 10 he would have 100% increase thats bit too op